frontend/src/components/Displays/LocalizationVerification/ScheduleMissionChecklistDialog.tsx

```diff
@@ -0,0 +1,256 @@
+    setIsCheckConfirmed: (b: boolean) => void
+    robot: Robot
+    deckName: string
+}
+
+const LocalisationDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+    const { installationDecks } = useInstallationContext()
+
+    const newDeck = installationDecks.find((deck) => deck.deckName === deckName)
+
+    return (
+        <>
+            <Dialog.Header>
+                <Typography variant="h5">{TranslateText('Confirm placement of robot')}</Typography>
+            </Dialog.Header>
+            <Dialog.Content>
+                <VerticalContent>
+                    <Typography>
+                        {`${robot.name} (${robot.model.type}) ${TranslateText(
+                            'needs to be placed on marked position on'
+                        )} ${deckName} `}
+                        <b>{TranslateText('before')}</b>
+                        {` ${TranslateText('clicking confirm')}.`}
+                    </Typography>
+                    {newDeck && newDeck.defaultLocalizationPose && (
+                        <DeckMapView deck={newDeck} markedRobotPosition={newDeck.defaultLocalizationPose}></DeckMapView>
+                    )}
+                    <HorizontalContent>
+                        <Checkbox
+                            crossOrigin={undefined}
+                            onChange={(e: ChangeEvent<HTMLInputElement>) => setIsCheckConfirmed(e.target.checked)}
+                        />
+                        <Typography>
+                            {`${TranslateText('I confirm that')} ${robot.name} (${robot.model.type}) ${TranslateText(
+                                'has been placed on marked position on'
+                            )} `}
+                            <b>{deckName}</b>
+                        </Typography>
+                    </HorizontalContent>
+                </VerticalContent>
+            </Dialog.Content>
+        </>
+    )
+}
+
+const PressureDialog = ({ setIsCheckConfirmed, robot, deckName }: CheckDialogProps) => {
+    const { TranslateText } = useLanguageContext()
+
+    let statusText = ''
+    if (robot.pressureLevel) {
+        if (
+            robot.pressureLevel > robot.model.lowerPressureWarningThreshold &&
+            robot.pressureLevel < robot.model.upperPressureWarningThreshold
+        ) {
+            statusText = `${TranslateText('The current pressure level is')} ${robot.pressureLevel} ${TranslateText(
+                'which is within the suggested range'
+            )}`
+        } else {
+            statusText = `${TranslateText('Warning')}: ${TranslateText('The current pressure level is')} ${
+                robot.pressureLevel
+            } ${TranslateText('which is NOT within the suggested range')}`
+        }
+    } else {
+        statusText = TranslateText(
+            'The pressure measurement is not currently available, so start the mission at your own risk.'
```

Comment:
This is funny but we can't have this text either ahaha i'll do a PR on this later today